config/locales/event_types/es.yml

```diff
@@ -11,6 +11,7 @@ es:
     email_changed: Email cambiado
     new_personal_key: Clave personal cambiado
     password_changed: Contraseña cambiada
+    personal_key_used: Clave personal utilizada para iniciar sesión
```

Comment:
@jgsmith-usds Is this an accurate translation?

Reply:
Looks reasonable to me.